lib/rbac/filterer.rb

```diff
@@ -208,6 +208,11 @@ def search(options = {})
       ids_clause             = nil
       target_ids             = nil
 
+      if options[:use_sql_view]
```

Comment:
@NickLaMuro thanks - your comments put this into the chopping block

Reply:
I do hope you are referring to `add_joins`, and not the line you have tagged here.  I think the `option[:use_sql_view]` still makes sense for the backport at least.

But just to be clear, as you mentioned in your other comment, if specs or even just replication steps in the PR description can make this reproducible in the `Service.yaml`, I find that completely fine.

Reply:
I removed this mucking with the `options` part - turns out we can continue with the add_select gook that has always been there

yes, we're keeping the `options[:use_sql_views]` but we don't need this mini block up front (that in all honesty, changed 4 other lines in this code but makes it look like it doesn't - which isn't so good)